cmd/dev/snapshots.cpp

```diff
@@ -252,11 +260,26 @@ void parse_command_line(int argc, char* argv[], CLI::App& app, SnapshotToolboxSe
 }
 
 //! Convert one duration into another one returning the number of ticks for the latter one
+//! \param elapsed the duration to convert
 template <typename D, typename R, typename P>
 auto duration_as(const std::chrono::duration<R, P>& elapsed) {
     return std::chrono::duration_cast<D>(elapsed).count();
 }
 
+//! Convert the given duration into milliseconds
+//! \param elapsed the duration to convert
+template <typename R, typename P>
+auto as_milliseconds(const std::chrono::duration<R, P>& elapsed) {
+    return duration_as<std::chrono::milliseconds>(elapsed);
+}
+
+//! Convert the given duration into seconds
+//! \param elapsed the duration to convert
+template <typename R, typename P>
+auto as_seconds(const std::chrono::duration<R, P>& elapsed) {
+    return duration_as<std::chrono::seconds>(elapsed);
+}
```

Comment:
After watching a [talk on chrono](https://www.youtube.com/watch?v=P32hvk8b13M) I believe that this is not a proper usage of duration_cast. A better approach is to create a `string format_duration(duration)`. I suggest to format it to seconds with ms fractional part e.g.: `12.455 sec`